terraform/testcases/prometheus_mock_awsprw/otconfig.tpl

```diff
@@ -0,0 +1,22 @@
+receivers:
+  prometheus:
+    config:
+      global:
+        scrape_interval: 15s
+      scrape_configs:
+      - job_name: "test-pipeline-job"
+        static_configs:
+        - targets: [ ${sample_app_listen_address_host}:${sample_app_listen_address_port} ]
+exporters:
+  awsprometheusremotewrite:
+    endpoint: "https://${mock_endpoint}"
+    aws_auth:
+      region: "us-west-2"
```

Comment:
Same question on why this region is fixed.

Reply:
Good question, I didn't write the original tests so I'm not sure why it's fixed, and same for above.